packages/react-components/src/components/utils/useLongPress.ts

```diff
@@ -69,10 +79,6 @@ export default function useLongPress(
   }
 
   function handleOnTouchEnd(): void {
-    if (isMobile) {
-      isLongPress ? onLongPress() : onClick();
-    }
-
     timerRef.current && clearTimeout(timerRef.current);
   }
 
```

Comment:
My gut reaction is the result of this hook should be memoized (which annoyingly would involve memoizing each of the functions in this hook)
... do you think it needs memoized?